Bliss.jmd

```diff
@@ -0,0 +1,168 @@
+Left figures: Bliss applied on pure cell number of data, i.e., the traditional approach.
+Right figures: Bliss applied on the model results, i.e., our approach for drug combination.
```

Comment:
Does this note match the figures?

Reply:
I meant to do it, but I thought I'd do it in another PR, since there are ~400 lines changed here.

Reply:
Should I add it to this PR?

Reply:
No, I like starting another PR. Just wanted to check. I'll approve this.